IssueTracker/IssueTracker.UnitTest/TestInitialization.cs

```diff
@@ -25,9 +26,32 @@ public static void AssemblyInitialize(TestContext tc)
             BaseClass.TestContext = tc;
 
             IConfigurationFactory cf = new ConfigurationFactory(configuration);
+            var blobConfig = cf.Create<IBolbConfigurationFactory>();
             var metadataconfig = cf.Create<IMetaDataConfiguration>();
+            BaseClass.s_blobData = new BolbData(blobConfig);
             BaseClass.s_metaDataProvider = new MetaData(metadataconfig);
 
+            //begin populate Blob Test
+            var path = "TextForTest.txt";
```

Comment:
You need to rethink this.

Reply:
<img width="437" alt="image" src="https://user-images.githubusercontent.com/74471664/190164253-783c54df-d035-4377-a2c7-a5de3266f703.png">